web/src/utils/uuidToURL.ts

```diff
@@ -0,0 +1,20 @@
+export const uuidToURL = (uuid: string, objectName: string) => {
+  // TODO: needs proper implementation
+  if (!uuid || !objectName) return "";
```

Comment:
Since you typed the params to not allow them to be undefined, this isn't necessary, right?

Reply:
Done!
You're right.